scalding-core/src/main/scala/com/twitter/scalding/Operations.scala

```diff
@@ -226,10 +310,20 @@ package com.twitter.scalding {
         evictions.increment(evicted.get.size)
       evicted
     }
+
+    def putAll(kvs: Map[K, V]): Option[Map[K, V]] = {
+      val (curHits, evicted) = summingCache.putWithHits(kvs)
+      misses.increment(1 - curHits)
```

Comment:
is this logic correct? I don't think so. `1` was the number of keys before. I think you need `kvs.size` instead of `1`